build/buildkite-build.sh

```diff
@@ -59,6 +59,9 @@ do
   sleep 1
 done
 
+echo "--- :node: Install jsdom"
+npm install jsdom
```

Comment:
@tpolecat I have to add this to run some of the tests. I don't know if there is a way to have this installed only once